### PR TITLE
Remove Mango's coverage package

### PR DESCRIPTION
**Pull request by jon-harmon-macmillan**
https://github.com/MangoTheCat/testCoverage isn't supported by Mango anymore. remove link?

### CODE REVIEW

**Comment by jimhester** [comment]
The docker based one is also no longer active, I think it is still useful to have them in the README as a reference, but maybe tweak the title of the section to indicated they are abandoned...

**Comment by jon-harmon-macmillan** [comment]
That makes sense. I'm not sure how/if I can make an additional change to something I just edited via github directly, though. Poking around doesn't show any easy way to keep editing in-github. Feel free to reject this PR, or I can tweak if I can figure out how...

File: README.md [hunk 1]
```diff
@@ -240,5 +240,4 @@ which _can_ modify behavior (usually due to package bugs which are masked with
 higher optimization levels).
 
 # Alternative Coverage Tools #
```
Comment (jon-harmon-macmillan) [inline README.md]:
```suggestion
# Alternative Coverage Tools #
- <https://github.com/MangoTheCat/testCoverage> (no longer supported)
```

File: README.md [hunk 2]
```diff
@@ -240,5 +240,5 @@ which _can_ modify behavior (usually due to package bugs which are masked with
...
-- <https://github.com/MangoTheCat/testCoverage>
+- <https://github.com/MangoTheCat/testCoverage> (no longer supported)
 - <http://r2d2.quartzbio.com/posts/r-coverage-docker.html>
```
Comment (jon-harmon-macmillan) [inline README.md]:
```suggestion
- <http://r2d2.quartzbio.com/posts/r-coverage-docker.html> (no longer supported)
```

**Comment by jon-harmon-macmillan** [comment]
How's that? I didn't want to put it in the title, in case you eventually want to add an alternative that IS supported.

**Comment by jimhester** [comment]
Great work, this package is now better thanks to you!

Thanks!! You are da bomb!

**Pull request merged**